mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -0,0 +1,105 @@
+CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
+	CLASS class_4467 NotAngryGoal
```

Comment:
Aren't all goals except the sting goal "not angry" goals? What makes this one different? Is it just a `MoveAroundGoal`?

Reply:
`NotAngryGoal` is abstract, and is extended by `EnterHiveGoal`, `GrowCropsGoal`, `FIndHiveGoal`, `PollinateGoal`, and `MoveToTargetGoal` (and by extension, `MoveToHiveGoal` and `MoveToFlowerGoal`).

`NotAngryGoal` only exists to make sure that those goals are only run when the bee is not angry. Therefore it makes sense for it to be `NotAngryGoal`.